compiler/rustc_error_codes/src/error_codes.rs

```diff
@@ -423,6 +423,7 @@ E0720: include_str!("./error_codes/E0720.md"),
 E0722: include_str!("./error_codes/E0722.md"),
 E0724: include_str!("./error_codes/E0724.md"),
 E0725: include_str!("./error_codes/E0725.md"),
+E0725: include_str!("./error_codes/E0726.md"),
```

Comment:
It's also *726* and not *725*.
```suggestion
E0726: include_str!("./error_codes/E0726.md"),
```